.github/workflows/ci.yaml

```diff
@@ -15,21 +15,82 @@ env:
   NODE_OPTIONS: --max_old_space_size=6144
   GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
+permissions: {}
```

Comment:
This is the default

Reply:
Only if selected as such in the org/repo settings, and actually I think the default is `contents: read` for the restricted setting.  Better to be explicit IMO.